.github/workflows/review.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: GPTcode-reviewer
-        uses: PierreGode/GPTcode-reviewer@main
+        uses: PierreGode/GPTcode-reviewer@PR-LOG
```

Comment:
Using a branch reference (PR-LOG) instead of a specific commit or tag may lead to instability in your workflow if the branch changes or is deleted.